incubator/node/templates/deployment.yaml

```diff
@@ -13,6 +13,20 @@ spec:
     metadata:
       labels:
         app: {{ template "fullname" . }}
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+            {
+                "name": "git-clone-app",
+                "image": "{{ .Values.image }}", 
+                "command": [ "/bin/sh", "-c" , "git clone {{ .Values.repository }} /app" ],
```

Comment:
can we move the chown and `npm install` to here? So we can keep the main container only a single command?